resources/views/processes/screens/import.blade.php

```diff
@@ -48,7 +48,8 @@
             <ul v-show="options" class="list-unstyled">
                 <li v-for="item in options">
                     <i :class="item.success ? 'fas fa-check text-success' : 'fas fa-times text-danger'"></i>
-                    @{{item.label}} - @{{item.message}}
+                    @{{item.label}} - @{{item.message}} 
+                    <span v-if="item.info" :class="'text-danger'"style="content:'\a';white-space:pre;"> @{{item.info}}.</span>
```

Comment:
Maybe you could use bootstrap class to jump the line and remove the style attr. If you use **white-space:pre** if the text is longer that the container it will render text out the container. Please see the screenshot. With d-block class we are using bootstrap and fixing the issue with long strings:

`<span v-if="item.info" :class="'text-danger d-block'"> @{{item.info}}.</span>`


**Issue using white-space issue:**
![Screen Shot 2022-07-29 at 11 18 44](https://user-images.githubusercontent.com/90727999/181780115-7ba202fc-e001-44d3-9a47-715f6b2d8251.png)


**Fix using d-block**
![Screen Shot 2022-07-29 at 11 19 12](https://user-images.githubusercontent.com/90727999/181780186-db795f0e-f7ef-4063-a0b9-be4de83aaf2c.png)

Reply:
suggestion applied